manifest-production.yml

```diff
@@ -0,0 +1,17 @@
+---
+inherit: base-manifest.yml
```

Comment:
Manifest inheritance was deprecated recently: https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#deprecated. The new recommendation is to use yaml anchors for DRY.